### PR TITLE
input/sdl_impl: fix rumble support on DualSense.

### PR DESCRIPTION
**Pull request by Zangetsu38**
# About:
- fix rumble on DualSense
- Dualsense return 78 value for using rumble, return 0 only if both frequency is 0, and after check function diferent value is normal

value return can be different 0, is not error is normal, error is only -1.

### CODE REVIEW

**Review by FernandoS27 — APPROVED** [review]

File: src/input_common/sdl/sdl_impl.cpp [hunk 1]
```diff
@@ -126,14 +126,12 @@ class SDLJoystick {
     }
 
     bool RumblePlay(u16 amp_low, u16 amp_high) {
-        constexpr u32 rumble_max_duration_ms = 1000;
```
Comment (Morph1984) [inline src/input_common/sdl/sdl_impl.cpp:129]: I'm curious, why was this removed?

Comment (Zangetsu38) [reply src/input_common/sdl/sdl_impl.cpp:129]: I restore later, apparently some game is stupid coded and no return 0 after rumble finish 🤦‍♂️🤷‍♂️

**Comment by Zangetsu38** [comment]
i have restored max duration, some stupid game is apparently bad coded and no send 0 in frequency for stop vibration after it finished

**Review by Morph1984 — APPROVED** [review]
